packages/cardano-serialization-lib/src/loadCardanoSerializationLib.ts

```diff
@@ -8,12 +6,11 @@ export const isNodeJs = (): boolean => {
   }
 };
 
-export type CardanoSerializationLib = typeof CardanoSerializationLibNodeJs;
+export type CardanoSerializationLib = typeof import('@emurgo/cardano-serialization-lib-nodejs');
 
 /**
- * Loads the environment-specific library.
- * The type of each complete library is the same, so one is statically imported for the return type.
- * Dynamically loads the browser library.
+ * Dynamically loads the environment-specific library.
+ * The type of each complete library is the same.
```

Comment:
This comment might be misleading, since we discovered there is a difference which could be a bug that we need to follow-up. Let's document the situation here

Reply:
I'll need to try to reproduce the original issue again. I've just tried to reproduce with a minimal test case and TypeScript seems to be happy about that:

```ts
type CardanoSerializationLib = typeof import('@emurgo/cardano-serialization-lib-nodejs');
type CardanoSerializationLibBrowser = typeof import('@emurgo/cardano-serialization-lib-browser');

let libNodejs: CardanoSerializationLib;
let libBrowser: CardanoSerializationLibBrowser;

const doSthWithLibNodejs = (lib: CardanoSerializationLib): void => {}
const doSthWithLibBrowser = (lib: CardanoSerializationLibBrowser): void => {}

doSthWithLibNodejs(libBrowser);
doSthWithLibBrowser(libNodejs);
```

Reply:
Great!